demo-spring-testing/src/main/java/com/examples/spring/demo/Example.java

```diff
@@ -0,0 +1,18 @@
+package com.examples.spring.demo;
+
+public class Example implements Cloneable {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a "clone()" method to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2157)